runtime/node/schemas/update-schema.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+SCHEMA_FILE=sdk.schema
+UISCHEMA_FILE=sdk.uischema
+BACKUP_SCHEMA_FILE=sdk-backup.schema
+BACKUP_UISCHEMA_FILE=sdk-backup.uischema
+
+echo "Running schema merge on nodejs runtime."
+[ -f "$SCHEMA_FILE" ] && mv "./$SCHEMA_FILE" "./$BACKUP_SCHEMA_FILE"
+[ -f "$UISCHEMA_FILE" ] && mv "./$UISCHEMA_FILE" "./$BACKUP_UISCHEMA_FILE"
+
+bf dialog:merge "*.schema" "!sdk-backup.schema" "*.uischema" "!sdk-backup.uischema" "!sdk.override.uischema" "../runtime/package.json" -o $SCHEMA_FILE
```

Comment:
this command will now also copy exported assets from installed components.   if not specified it will write the copied files to a local folder, NOT into the dialogs/imported folder that Composer uses.

Consider adding the `--import dialogs/imported`

Reply:
Added `../dialogs/imported` as the destination folder of imported assets.